package pl.lodz.p.it.carrental.misc.exceptions;

public class LoginAlreadyTaken extends Exception {
}
